### DISCUSSION TITLE
LSP4IJ 0.5.0 Release

### DISCUSSION THREAD

**Discussion by angelozerr** [redhat-developer/lsp4ij#509] [Announcements]
Hi,

Because of this major bug issue https://github.com/redhat-developer/lsp4ij/issues/504 which breaks the standard `Find Usages`, we are about to release LSP4IJ 0.5.0. Highlights of this release include several bug fixes (for semantic tokens, signature help and fix for Find Usages etc)
 
 More details in the <a href="https://github.com/redhat-developer/lsp4ij/milestone/5?closed=1">changelog</a>.
 
 Please give a try to the [latest nightly build](https://plugins.jetbrains.com/plugin/23257-lsp4ij/versions/nightly).

 Unless you report a blocking issue, we'll proceed with the release next Monday 09/09/2024. @aparnamichael @turkeylurkey @mrglavas it is just bug fixes which should not impact features that you are using (signature help, semantic tokens), but if you needmore time we can delay it to Tuesday if you wish.
 
 cc @CppCXY @InSyncWithFoo @seachicken @KUGDev @AlexWeinstein92 @aparnamichael @MituuZ @turkeylurkey @mrglavas @koxudaxi @FalsePattern @onriv @kristofvb

**Comment by angelozerr** [comment]
@aparnamichael @turkeylurkey @mrglavas I will try to fix your issue https://github.com/OpenLiberty/liberty-tools-intellij/issues/897 to embed it to the 0.5.0 release

**Reply by mrglavas** [reply]
I responded on the issue, but also confirming here that the fix works. Thanks.

**Reply by angelozerr** [reply]
Great! Thanks for your feedback 0.5.0 release is reviewing by JetBrains.

**Comment by angelozerr** [comment]
LSP4IJ 0.5.0 il now available in marketplace.

Hope you will enjoy with it.
